### PR TITLE
answers default class and test class

### PR DESCRIPTION
**Pull request by deprilch**

### CODE REVIEW

File: src/main/java/com/onseo/courses/ds/answers/Answers.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package com.onseo.courses.ds.answers;
+
+public class Answers {
+
+    private String TESTED_USER_ID;
```
Comment (MaksymChervinskyi) [inline src/main/java/com/onseo/courses/ds/answers/Answers.java]: parameters and members must be written from lower case letter

File: src/main/java/com/onseo/courses/ds/answers/Answers.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    private String QUESTION_ID;
+    private String[] ANSWERS_IDS;
+
+    public Answers(){
+
+    }
+
+    public void setTESTED_USER_ID(String id){
```
Comment (MaksymChervinskyi) [inline src/main/java/com/onseo/courses/ds/answers/Answers.java]: use camelCase

Comment (deprilch) [reply src/main/java/com/onseo/courses/ds/answers/Answers.java]: done

File: src/main/java/com/onseo/courses/ds/answers/Answers.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package com.onseo.courses.ds.answers;
+
+public class Answers {
+
+    private String tested_user_id;
```
Comment (MaksymChervinskyi) [inline src/main/java/com/onseo/courses/ds/answers/Answers.java]: Rename this to testedUserId and other like my example

**Pull request merged**